apps/vscode-extension/package.json

```diff
@@ -31,6 +31,24 @@
           "type": "string",
           "default": "",
           "description": "If not empty, will add a prefix to all the generated variables"
+        },
+        "otter.styling.variable.categories": {
```

Comment:
Why not relying on Stylelint as it is done for eslint?

Reply:
As discuss we will keep it like that